[Docs] Update README and encourage contributions

paramaggarwal opened the pull request.

paramaggarwal committed:
Update README.md

brody2consult commented on README.md:
```diff
@@ -1,7 +1,5 @@
 # electrino
 
-:warning: **Do not use this in production. This project is in its early stages** :warning:
```
I would favor keeping the warning with only the second sentence, maybe with some extra info. Idea:

```
:warning: __This project is in its early stages and contributions would be much appreciated.__ :warning:
```

paramaggarwal replied:
Good idea - let's be explicit about what we need help with. "Port more APIs from Electron into this project"?

paramaggarwal replied:
Please check now.

brody2consult commented on README.md:
```suggestion
Cross-platform support | Yes | Limited to macOS and Windows 10 at this time
```

brody2consult reviewed (commented):
I would favor making the title a little more generic, for example: update README.md & encourage more contributions

amilajack commented:
Agree with @brodybits

paramaggarwal committed:
Update README.md

Co-Authored-By: Chris Brody <chris.brody+brodybits@gmail.com>

paramaggarwal commented:
Makes sense. I have updated PR title and also added mention of Windows as suggested. The only thing pending now is to make explicit mention of what people can contribute.

amilajack commented:
These are the two main areas that I think we need contributions from:

* Implementing require
* Embedding node

Only a small subset of electron API's are actually used when compared to the usage of nodejs by electron apps.

paramaggarwal committed:
Update README.md

paramaggarwal commented:
I have updated the doc. Feel free to make further edits by modifying my PR. Thanks.

paramaggarwal commented:
Just came across https://github.com/mceSystems/node-jsc - I think this is our answer.

amilajack commented:
Good find! We'll also need something like this for windows as well

styfle commented:
Looks like `node-jsc` was last updated in October of 2018, so probably not the best idea to adopt it for a new project. It's two Node LTS releases behind so many packages on npm already dropped support for Node 8.
